addon/styles/addon.css

```diff
@@ -0,0 +1,3 @@
+.file-upload input[hidden] {
+  display: none !important;
+}
```

Comment:
ember-cli automatically appends this file to `vendor.css`, so we don't need to change any configuration.